interfaces/Libraries.ts

```diff
@@ -155,6 +161,17 @@ const sentenceTransformers = (model: ModelData) =>
 
 model = SentenceTransformer("${model.modelId}")`;
 
+const spacy = (model: ModelData) =>
+`# pip install https://huggingface.co/${model.modelId}/resolve/main/${nameWithoutNamespace(model.modelId)}-any-py3-none-any.whl
```

Comment:
this is maybe not visible enough? Should it be not a comment as it's the first required step?

Reply:
I put it as a comment so people could copy paste the code and it would by Python right away. I agree it's not visible enough.

What syntax would you suggest to use? `!pip install` Or simply uncommented pip install?

Reply:
up to you, i think both would work.

in the future maybe we need to allow multiple code snippets per library, with different highlighting... might be overly complex though

Reply:
Thanks for your input!
I resolved the issues in https://github.com/huggingface/huggingface_hub/pull/60